selection.py

```diff
@@ -0,0 +1,24 @@
+def sort(array: list[int]) -> list[int]:
+    """
+    Sorting a list by selection (O(n²) complexity)
+    """
+    for start in range(len(array)-1):
```

Comment:
Le -1 ne doit pas être nécessaire
Par exemple range(5) itère de 0 à 4

Reply:
Ce n'est pas la peine de trier le dernier nombre tout seul, d'où le -1.

Reply:
En effet !